nrlf-dependencies.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+while read dep; do
+    lib="${dep% *}"
+    "asdf plugin add ${lib}"
```

Comment:
I had to change this to:

```suggestion
    asdf plugin add ${lib}
```

on macOS